operator/internal/controller/pulumi/flux.go

```diff
@@ -70,37 +72,28 @@ func checksumOrDigest(source unstructured.Unstructured) (string, error) {
 	return checksum, nil
 }
 
-// checkFluxSourceReady looks for the conventional "Ready" condition to see if the supplied object
-// can be considered _not_ ready. It returns an error if it can determine that the object is not
-// ready, and nil if it cannot determine so.
-func checkFluxSourceReady(obj unstructured.Unstructured) error {
+func checkFluxSourceReady(obj *unstructured.Unstructured) bool {
```

Comment:
Why return bool instead of error? Seems like we lose some debugability?

Reply:
The error was simply being used to convey a singular message, and there was need to call this method from the mapper function (which cannot really handle errors).  For those reasons, I simplified.